apps/api/src/server/applications/application/documents/document.routes.js

```diff
@@ -233,6 +234,26 @@ router.get(
 	asyncHandler(getDocumentProperties)
 );
 
+router.get(
+	'/document/:guid/properties',
```

Comment:
Is there a reason that "document" is singular? generally rest-like APIs use plural names, and this should be consistent with the other endpoints, e.g. `/applications/{id}/documents/{guid}/properties`

Looks like we're inconsistent here, so might be worth making it a separate PR to sort.